.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,8 +22,8 @@ the requirements below.
 
 - [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
 - [ ] PR description included
-- [ ] Relevant documentation is changed or added (and PR referenced)
 - [ ] `yarn test` passes and tests are updated/added
-- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
+- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
```

Comment:
Added this but not sure whether we want to enforce this for commit messages?

Reply:
Can you check if we have a backlog item to enforce? If not, let's create one

Reply:
We have one already!